JSqlServerBulkInsert/src/test/java/de/bytefish/jsqlserverbulkinsert/test/base/TransactionalTestBase.java

```diff
@@ -16,7 +16,8 @@
 
     @Before
     public void setUp() throws Exception {
-        connection = DriverManager.getConnection("jdbc:sqlserver://localhost:1433;databaseName=TestDatabase", "philipp", "test_pwd");
+        //connection = DriverManager.getConnection("jdbc:sqlserver://localhost:1433;databaseName=TestDatabase", "philipp", "test_pwd");
+        connection = DriverManager.getConnection("jdbc:sqlserver://localhost;instanceName=MSSQLSERVER2017;databaseName=TestDatabase;", "philipp", "test_pwd");
```

Comment:
Are we locking ppl to a specific Sql Server version?

Reply:
Also, maybe the db should be called something more specific to this project, bc what if ppl test other projects that also have a TestDatabase.

Reply:
I knew you would spot this. 👍 I will add a configuration file to set the Connection String,

Reply:
Done!